apriltag_ros/config/tags.yaml

```diff
@@ -19,6 +19,14 @@
 #   ]
 standalone_tags:
   [
+   {id: 0, size: 0.086},
+   {id: 1, size: 0.130},
+   {id: 2, size: 0.043},
+   {id: 3, size: 0.086},
+   {id: 4, size: 0.086},
+   {id: 5, size: 0.086},
+   {id: 6, size: 0.086},
+   {id: 7, size: 0.086},
```

Comment:
This change looks project specific.  It's unlikely you'd want every user of apriltags_ros in the world to have this change, so please make this locally, and not in apriltags_ros